controllers/front/ProductController.php

```diff
@@ -806,14 +806,16 @@ public function getBreadcrumbLinks()
     {
         $breadcrumb = parent::getBreadcrumbLinks();
 
-        $categoryDefault = new Category($this->product->id_category_default);
+        $categoryDefault = new Category($this->product->id_category_default, $this->context->language->id);
```

Comment:
Please note that casts should be done for object models, have a look at https://github.com/PrestaShop/PrestaShop/pull/5980

This happens because of https://github.com/PrestaShop/PrestaShop/blob/develop/classes/ObjectModel.php#L218-L220